idea/indexer/src/common/helpers/get-payload-by-gear-event.ts

```diff
@@ -57,26 +57,25 @@ function userMessageReadPayload({ id, reason }: UserMessageReadData): UserMessag
 }
 
 function programChangedPayload({ id, change }: ProgramChangedData): ProgramChangedInput {
-  const res = { id: id.toHex(), programStatus: ProgramStatus.UNKNOWN };
-
-  if (change.isActive) {
-    res.programStatus = ProgramStatus.ACTIVE;
-  }
-
-  if (change.isInactive) {
-    res.programStatus = ProgramStatus.EXITED;
-  }
-
-  if (change.isPaused) {
-    res.programStatus = ProgramStatus.EXITED;
-  }
+  const status = change.isActive
```

Comment:
IMHO with `if` statement is easier to read instead of the steps `ternary operations`. Of course, it's up to you to refactor this part or not.

Reply:
These conditions are small, so I think it's okay in this case.